### PR TITLE
loggin per OS

### PR DESCRIPTION
**Pull request by rssk**
Logging is borked when packaged, push logs out to OS specific dirs

### CODE REVIEW

File: packages/coinstac-ui/scripts/build.js [hunk 1]
```diff
@@ -10,7 +10,6 @@ const options = {
   overwrite: true,
   platform: 'all',
   prune: true,
-  version: '1.1.1',
```
Comment (rssk) [inline packages/coinstac-ui/scripts/build.js:13]: This is automatically detected by `packager`, setting it explicitly is just another thing to update

Comment (swashcap) [inline packages/coinstac-ui/scripts/build.js:13]: :+1:

File: packages/coinstac-ui/app/main/utils/boot/configure-logger.js [hunk 2]
```diff
@@ -1,16 +1,28 @@
...
 const app = require('ampersand-app');
+const path = require('path');
 const cliOpts = require('app/main/utils/boot/parse-cli-input.js').get();
 const config = require('../../../config.js');
 const winston = require('winston');
+const promisify = require('bluebird').promisify;
+const mkdirp = promisify(require('mkdirp'));
+
+const resolveHome = (filepath) => {
+  return filepath.replace(/~|\$HOME/, process.env.HOME);
```
Comment (swashcap) [inline packages/coinstac-ui/app/main/utils/boot/configure-logger.js:12]: Eh. Seems sketch.

File: packages/coinstac-ui/app/config.js [hunk 3]
```diff
@@ -38,6 +38,13 @@ const conf = convict({
     noURLPrefix: true
   },
   logFile: 'coinstac-log.json',
+  logLocations: {
```
Comment (swashcap) [inline packages/coinstac-ui/app/config.js:41]: Not sure that we _need_ configurable log locations. Might be needlessly complex. We can address at another time.

**Comment by swashcap** [comment]
:+1: Couple issues, we can address when we overhaul config.

**Pull request merged**